runners/google-cloud-dataflow-java/arm/build.gradle

```diff
@@ -0,0 +1,159 @@
+def dockerTag = project.findProperty('docker-tag') ?: new Date().format('yyyyMMddHHmmss')
+ext.DockerJavaMultiarchImageName = "${DockerJavaMultiarchImageContainer}:${dockerTag}"
+
+def runnerV2PipelineOptionsARMARM = [
+  "--runner=TestDataflowRunner",
+  "--project=${dataflowProject}",
+  "--region=${dataflowRegion}",
+  "--tempRoot=${dataflowValidatesTempRoot}",
+  "--sdkContainerImage=${DockerJavaMultiarchImageContainer}:${dockerTag}",
+  "--experiments=use_unified_worker,use_runner_v2",
+  "--firestoreDb=${firestoreDb}",
+  "--machine_type=t2a-standard-1",
+]
+
+// Build and push multi-arch docker images to a container registry for use within tests.
+// NB: Tasks which consume docker images from the registry should depend on this
+// task directly ('dependsOn buildAndPushDockerJavaMultiarchContainer'). This ensures the correct
+// task ordering such that the registry doesn't get cleaned up prior to task completion.
+def buildAndPushDockerJavaMultiarchContainer = tasks.register("buildAndPushDockerJavaMultiarchContainer") {
+  def javaVer = "java8"
+  if(project.hasProperty('compileAndRunTestsWithJava17')) {
+    javaVer = "java17"
+  } else if(project.hasProperty('compileAndRunTestsWithJava11')) {
+    javaVer = "java11"
+  }
+  dependsOn ":sdks:java:container:${javaVer}:docker"
+}
+
+// Clean up built Java images
+def cleanUpDockerJavaMultiarchImages = tasks.register("cleanUpDockerJavaMultiarchImages") {
+  doLast {
+    exec {
+      commandLine "gcloud", "--quiet", "container", "images", "untag", "${DockerJavaMultiarchImageName}"
+    }
+    exec {
+      commandLine "./scripts/cleanup_untagged_gcr_images.sh", "${DockerJavaMultiarchImageContainer}"
+    }
+  }
+}
+
+afterEvaluate {
+  // Ensure all tasks which use published docker images run before they are cleaned up
+  tasks.each { t ->
+    if (t.dependsOn.contains(buildAndPushDockerJavaMultiarchContainer) && !t.name.equalsIgnoreCase('printrunnerV2PipelineOptionsARM')) {
+      t.finalizedBy cleanUpDockerJavaMultiarchImages
+    }
+  }
+}
+
+task printrunnerV2PipelineOptionsARM {
+  dependsOn buildAndPushDockerJavaMultiarchContainer
+
+  doLast {
+    println "To run a Dataflow job with runner V2 on ARM, add the following pipeline options to your command-line:"
+    println runnerV2PipelineOptionsARM.join(' ')
+    println "Please delete your image upon completion with the following command:"
+    println "gcloud container images delete --force-delete-tags ${DockerJavaMultiarchImageName}"
+  }
+}
+
+task examplesJavaRunnerV2IntegrationTestARM(type: Test) {
+  group = "Verification"
+  dependsOn buildAndPushDockerJavaMultiarchContainer
+
+  systemProperty "beamTestPipelineOptions", JsonOutput.toJson(runnerV2PipelineOptionsARM)
+
+  // The examples/java preCommit task already covers running WordCountIT/WindowedWordCountIT so
+  // this postCommit integration test excludes them.
+  include '**/*IT.class'
+  exclude '**/WordCountIT.class'

```

Comment:
These exclusions are because wordcount on dataflow already run as precommit, so example postcommit excluded them. Here may include WordCountIT, WindowedWordCountIT also

Reply:
sorry I mean to run wordcount we can just remove the two exclude statements (no need double include)